Applications/spire/include/spire/charting/chart_view.hpp

```diff
@@ -91,7 +85,7 @@ namespace Spire {
       std::vector<ChartValue> m_x_axis_values;
       int m_x_axis_text_width;
       std::vector<ChartValue> m_y_axis_values;
-      QtPromise<std::vector<Spire::Candlestick>> m_loaded_data;
+      QtPromise<std::vector<Spire::Candlestick>> m_loaded_candlestick_promise;
```

Comment:
The problem with the name is that nothing has been loaded. A promise is a data structure that performs an action at some unspecified point in the future but you have named it using the past tense as if this variable already loaded the candlesticks. You can name this ```pending_candlesticks```, ```candlestick_promise```, ```candlestick_loader``` etc... but don't use the past tense.